extensions/wikia/AdEngine/tests/AdEngine2ContextServiceTest.php

```diff
@@ -45,84 +45,84 @@ private function mockMonetizationModule() {
 	public function adContextDataProvider() {
 		$defaultParameters = [
 			'titleMockType' => 'article',
-			'flags' => [],
-			'expectedOpts' => [],
+			'flags' => [ ],
+			'expectedOpts' => [ ],
 			'expectedTargeting' => [ 'newWikiCategories' => [ 'test' ] ],
-			'expectedProviders' => [],
+			'expectedProviders' => [ ],
 			'expectedForceProviders' => null,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "expectedForceProviders" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)